gossip/node.h

```diff
@@ -42,6 +43,7 @@ typedef struct node_s {
   neighbor_t* neighbors;
   rw_lock_handle_t neighbors_lock;
   tips_cache_t tips;
+  uint8_t features;
```

Comment:
Can you move this to the API ? Some of the features are not node-related.

Reply:
so I should move this to `ciri/api/api.c`?

Reply:
Yes please